docs/spec.md

```diff
@@ -3324,23 +3363,25 @@ Performs element-wise max operation on tensors `lhs` and `rhs` and produces a
   Imposing an ordering on complex numbers involves surprising semantics,
   so in the future we are planning to remove support for complex numbers
   for this operation ([#560](https://github.com/openxla/stablehlo/issues/560)).
+* For quantized types:
+  * `dequantize_op_quantize(maximum, lhs, rhs, type(result))`.
 
 #### Inputs
 
-| Label | Name  | Type   | Constraints |
-|-------|-------|--------|-------------|
-| (I1)  | `lhs` | tensor | (C1)        |
-| (I2)  | `rhs` | tensor | (C1)        |
+| Label | Name  | Type                                  | Constraints |
+|-------|-------|---------------------------------------|-------------|
+| (I1)  | `lhs` | tensor or per-tensor quantized tensor | (C1)        |
+| (I2)  | `rhs` | tensor or per-tensor quantized tensor | (C1)        |
 
 #### Outputs
 
-| Name     | Type   | Constraints |
-|----------|--------|-------------|
-| `result` | tensor | (C1)        |
+| Name     | Type                                  | Constraints |
+|----------|---------------------------------------|-------------|
+| `result` | tensor or per-tensor quantized tensor | (C1)        |
 
 #### Constraints
 
-* (C1) `type(lhs) = type(rhs) = type(result)`.
+* (C1) `baseline_type(operand) = baseline_type(result)`.
```

Comment:
It looks like for a number of ops, "type(lhs) = type(rhs) = type(result)" got accidentally replaced with "baseline_type(operand) = baseline_type(result)".

Reply:
fixed.